x-pack/plugins/alerting/server/task_runner/action_scheduler/action_scheduler.ts

```diff
@@ -183,11 +180,7 @@ export class ActionScheduler<
     enqueueOptions: EnqueueExecutionOptions;
     bulkScheduleRequest: EnqueueExecutionOptions[];
   }) {
-    if (
```

Comment:
This doesn't seem quite right.  If for some reason `this.context.taskRunnerContext.supportsEphemeralTasks` becomes true, I think we'd end up scheduling all the tasks with it, whereas before we would have only done `context.taskRunnerContext.maxEphemeralActionsPerRule` of them.

I wonder if we should either remove queueing these as emphermeral completely here, or something else.  @mikecote ?

Reply:
Oh actually this is my mistake, we have @JiaweiWu who's working on making this setting no-op in https://github.com/elastic/kibana/pull/197421. Perhaps we can leave this one out from this PR. Sorry about that!

Reply:
okay sounds good, thanks!

Reply:
Resolved in this commit 375e71edb14c8b8c9d799f1cf16985fe230b0d15